### DISCUSSION TITLE
Tried calling start() Before the ready event.

### DISCUSSION THREAD

**Discussion by SuperStark** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#385] [Q&A]
I am seeing this missing message when trying to set this up:

Unexpected Exception Occured
An unexpected error occurred.
Error: Tried calling start() before the ready event!

I have the auth set up, google gives me the secret key to enter, but the above error gets in the way, if I click ignore then try to paste the key in, the submit button does not work. 

Also, Occurred has been misspelt :)

**Comment by SuperStark** [comment]
Scratch that I can see you are already aware, I hate the layout of github rn..

**Comment by Digitalcreateit** [comment]
I have the same error, how you fix it?

**Comment by sumeshir26** [comment]
@Digitalcreateit find the fix [in the FAQs](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/wiki/Frequently-Asked-Questions-(FAQ)#why-does-the-get-token-screen-fail-saving-tokens).

**Comment by im-coder-lg** [comment]
> I am seeing this missing message when trying to set this up:
> 
> Unexpected Exception Occured
> An unexpected error occurred.
> Error: Tried calling start() before the ready event!
> 
> I have the auth set up, google gives me the secret key to enter, but the above error gets in the way, if I click ignore then try to paste the key in, the submit button does not work. 
> 
> 
> Also, Occurred has been misspelt :) 

Two solutions according to FAQ.
- 1. Build the project via `npm`
- 2. Deactivate `Enable mic on startup` in settings, authenticate and reenable if you want.

**Comment by arvinoids** [comment]
This happened to me when I tried using the same OAuth code on another computer. To solve this, I had to reinitialize the app. You can do this by going to the Settings and scrolling to Application data Directory and then click on Open App Data Folder. You can delete the entire folder so that the next time you start the app, it will run the initial setup. This time I registered a new device so it doesn't conflict with my other one. Good luck!
